.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,7 +32,7 @@ GitHub Issue: #
 ### Always applicable
 No matter your changes, these checks always apply.
 - [ ] Your conventional commits are aligned with the **Impact on version** section.
-- [ ] Updated [CHANGELOG.md](../CHANGELOG.md).
+- [ ] Updated [CHANGELOG.md](../blob/main/CHANGELOG.md).
```

Comment:
I'm not sure this works? I tested by trying the link in your branch directly.

Reply:
uhmm... how i did it was comparing to the changelog file link and the link from the current PR template and that was the only part missing in the URL :/ 
the weird part is that those folders don't exists in the proj 
@jeanplevesque

Reply:
I think this is just saying that we're looking at the changelog of the main branch, and this change does work you can look at this draft pr: https://github.com/nventive/UnoApplicationTemplate/pull/409 I applied the changes from Ariel and it leads you to the correct place.

Reply:
ok